rust_parser/BUILD.bazel

```diff
@@ -6,31 +6,36 @@ rust_library(
     name = "parser",
     srcs = ["parser.rs"],
     visibility = [":__subpackages__"],
-    deps = ["//3rdparty/crates:syn"],
+    deps = ["@crates//:syn"],
 )
 
 rust_library(
     name = "lockfile_crates",
     srcs = ["lockfile_crates.rs"],
     visibility = [":__subpackages__"],
     deps = [
-        "//3rdparty/crates:cargo-lock",
         "//proto:messages_rust_proto",
+        "@crates//:cargo-lock",
         "@rules_rust//crate_universe:cargo_bazel",
     ],
 )
 
+# keep
+# For some reason Gazelle is adamant about transforming
+# @rules_rust_protobuf//3rdparty/crates:protobuf into
+# @rules_rust//proto/protobuf/3rdparty/crates:protobuf, which breaks the
+# build.
```

Comment:
That's because of this: https://github.com/Calsign/gazelle_rust/blob/b504c88bd96f553af373579c341107fe8ecb862b/BUILD.bazel#L54